### PR TITLE
Create scenario readme

### PR DESCRIPTION
**Pull request by christianbrb**
Fixes: #4570 

## Description

See issue #4570

## PR review checklist

The quality checklist that cannot be automatically verified.

- [ ] Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
- [ ] Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- [ ] Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- [ ] Regression test for the bug
    - [ ] Properly covers the bug
    - [ ] If an integration test is used, it could not be written as a unit test
- [ ] Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits

### CODE REVIEW

File: raiden/tests/scenarios/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+## Scenarios
+
+The scenarios listed within this folder are getting executed nightly by the [scenario player](https://github.com/raiden-network/scenario-player/) or can be executed locally after installing the scenario player.
+
+## Purpose
+
+The scenarios serve the following purpose
+
+- Full End-to-End Tests covering contracts, transport, Raiden client, blockchain, eth-client, monitoring service, pathfinding service and service contracts
+- Testing on a "dirty" blockchain
```
Comment (Dominik1999) [inline raiden/tests/scenarios/README.md]: very nice! I like your concise style

File: raiden/tests/scenarios/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- User Acceptance tests
```
Comment (Dominik1999) [inline raiden/tests/scenarios/README.md]: Why do you write Acceptance in capital and tests not?

File: raiden/tests/scenarios/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Scenarios
+
+#### [ms1_simple_monitoring](./ms1_simple_monitoring.yaml)
+
+A channel between two nodes is opened, a transfer is made. Then, node 1 goes offline and node 0 closes the channel. The last assert checks whether the monitoring service interferes and received its reward.
+
+#### [ms2_simple_monitoring](./ms2_simple_monitoring.yaml)
+
+A channel between two nodes is opened, a transfer is made. Then, node 1 goes offline and node 0 closes the channel. After the monitoring trigger block is passed node1 gets back online.
```
Comment (Dominik1999) [inline raiden/tests/scenarios/README.md]: maybe add, that "node 1 does not get back online in time and cannot update the channel info by herself." the monitoring services interferes and gets its reward

File: raiden/tests/scenarios/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+#### [ms3_simple_monitoring](./ms3_simple_monitoring.yaml)
+
+A channel between two nodes is opened, a transfer is made. Then, node 1 goes offline and node 0 closes the channel. Before the monitoring trigger block is passed node1 gets back online. Node1 should be able to call close itself and no reward for the MS.
```
Comment (Dominik1999) [inline raiden/tests/scenarios/README.md]: node1 after "passed" - there is a space missing. And "Node1" at the beginning of the next sentence

maybe don't use conjunctive mood for the last sentence. "Node 1 calls the smart contract itself (REMARK: she does not call close) and therefore the MS does not get triggered"

File: raiden/tests/scenarios/README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+#### [pfs1_get_a_simple_path](./pfs1_get_a_simple_path.yaml)
```
Comment (Dominik1999) [inline raiden/tests/scenarios/README.md]: we need to rewrite the pfs scenarios anyway. So let's keep it like that for now

**Comment by christianbrb** [comment]
@Dominik1999 Thanks :-) Applied all requested changes :-)

File: raiden/tests/scenarios/README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+A channel between two nodes is opened, a transfer is made. Then, node 1 goes offline and node 0 closes the channel. The last assert checks whether the monitoring service interferes and received its reward.
+
+#### [ms2_simple_monitoring](./ms2_simple_monitoring.yaml)
+
+A channel between two nodes is opened, a transfer is made. Then, node 1 goes offline and node 0 closes the channel. node 1 does not get back online in time and cannot update the channel info by herself. The monitoring services interferes and gets its reward. After the monitoring trigger block is passed node 1 gets back online.
+
+#### [ms3_simple_monitoring](./ms3_simple_monitoring.yaml)
+
+A channel between two nodes is opened, a transfer is made. Then, node 1 goes offline and node 0 closes the channel. Before the monitoring trigger block is passed node 1 gets back online. Node 1 calls the smart contract itself (REMARK: she does not call close) and therefore the MS does not get triggered.
```
Comment (Dominik1999) [inline raiden/tests/scenarios/README.md]: the REMARK was just for you. hehe

**Pull request merged**